.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: ruby
 rvm:
   - "2.3"
-  - "jruby"
+  - "jruby-9.1.17.0"
```

Comment:
Should we use jruby-1.17 and not 9.x? Before 2.3 this was jruby-1.7.25.

Reply:
I'm not sure that jruby-1.17 is a thing. It looks like it was you who removed the `jruby-1.7.25` in favour of just `jruby` back in May so we've been `9.<latest>` since then ;)

It looks like 1.7.27 is the latest non 9.x version but it's pretty old. Not sure why we would actively want to be using it, although I'm not familiar with the difference between the 1.x and 9.x versions.

Reply:
It depends when Travis CI updated to use 9.x of JRuby. I think it was using 1.19 at the time as 1.17 had an ssl issue and was deprecated.

I was thinking more for backwards compatibility (for now) that maybe it was best to just increase these slightly until it was working. To me, it seems like going from 2.1 to 2.5 in Ruby.

For now, as this is a list of Rubies, can we add jruby-1.19 (or something in the 1.x range), jruby 9000, 2.3 and 2.5 so we can get this running against different versions.

Reply:
Yeah, as you say it's just a list so we can just add some older ones to be sure it's still working on an older version. `jruby-9.1.17.0` is a ruby 2.3 implementation though so I think it makes sense  as it is if we've pinned the ruby version to 2.3 to also use a 2.3 jruby version